prow/cluster/starter.yaml

```diff
@@ -0,0 +1,247 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: plugins
```

Comment:
Document this file a bit? what is data: plugins? What would I do next?

Reply:
I'm leaving this until the "Next steps" section, which I'll fill in in this PR tomorrow.